oauth2withjwt/src/main/java/com/brother/oauth2withjwt/entity/RoleEntity.java

```diff
@@ -0,0 +1,28 @@
+package com.brother.oauth2withjwt.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "role")
+@Getter
+@Setter
+public class RoleEntity implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+    @Column(name = "role_name", unique = true, nullable = false)
```

Comment:
here is the entity and class name id role.  Consequently, another variable name is roleName is looking not good to me

Reply:
done